tests/integration/004_template/expected_outputs.json

```diff
@@ -1,8 +1,9 @@
 {
-    "tested2_template_type": "terraform",
-    "tested2_template_name": "tested1-",
-    "tested2_template_repository": "https://github.com/env0/templates",
-    "tested1_template_repository": "https://gitlab.com/env0/gitlab-vcs-integration-tests.git",
-    "tested2_template_path": "second",
-    "tg_tg_version" : "0.35.0"
+    "tested1_template_type": "terraform",
+    "tested1_template_name": "tested1-",
+    "tested1_template_repository": "https://github.com/env0/templates",
+    "tested2_template_repository": "https://gitlab.com/env0/gitlab-vcs-integration-tests.git",
+    "tested1_template_path": "second",
```

Comment:
The `data`s were mixed up 🤦🏻 
`tested1` pointed at `tested2` and vice-versa

Reply:
Maybe change the names to something meaningful instead?
Will probably be less confusing than 1 and 2

Reply:
thank God (and you of course), it wasn't just in my head... because it passed before I believed it was intentional.

Reply:
https://github.com/env0/terraform-provider-env0/pull/322/commits/70d3fe260af5383ccc86f38cbb04d3809a4bbb92